docs/OV_Runtime_UG/migration_ov_2_0/preprocessing.md

```diff
@@ -0,0 +1,68 @@
+# Preprocessing {#openvino_2_0_preprocessing}
+
+## Introduction
+
+Inference Engine API has preprocessing capabilities which are built on top of model expressed as `ngraph::Function` (now, it's `ov::Model`). One of the problems with it - preprocessing operations are not a part of the main model graph, so they are stored and executed separatelly:
```

Comment:
Suggesting to just describe old and new behavior, not stating that 'old API had problems' 
```suggestion
Inference Engine API has preprocessing capabilities which are built on top of model expressed as `ngraph::Function` (now, it's `ov::Model`). Before OpenVINO Runtime API 2.0 preprocessing operations were not a part of the main model graph, so they were stored and executed separately:
```